manifest.webapp

```diff
@@ -2,24 +2,15 @@
   "name": "Collect",
   "slug": "collect",
   "icon": "cozy_collect.svg",
-  "short_description": "Configuration application for konnectors",
-  "long_description": "__Collect__ will allow you to grab all your informations that you have using tons on services on providers websites or applications.\nEnjoy this new way to get control on your daily data :tada:",
-  "changes": "### New features :rocket:\n\nConnectors for you bank accounts!!:bank:",
   "category": "cozy",
   "source": "https://github.com/cozy/cozy-collect.git@build",
   "editor": "Cozy",
   "developer": {
     "name": "Cozy",
     "url": "https://cozy.io"
   },
-  "locales": {
-    "fr": {
-      "short_description": "Application de configuration pour les konnectors",
-      "long_description": "__Collect__ vous permettra de récupérer toutes vos informations que vous générez en utilisant la multitude de services en ligne.\nProfitez de cette nouvelle manière de récupérer le contrôle sur vos données quotidiennes :tada:",
-      "changes": "### Nouveautés :rocket:\n\nDes connecteurs pour vos comptes bancaires!!:bank:"
-    }
-  },
-  "langs": ["en", "fr"],
+  "locales": {},
+  "langs": [],
```

Comment:
It feels weird to have empty properties here. Is it possible to avoid them ?

Reply:
It's completely possible but I thought that could mean something like "the manifest will get `locales`/`langs` fields but filled during the build". Without them it could be a bit confusing, we could think that collect will never have fields `locales` and `langs` in its manifest.

Reply:
For me it's weird to add empty properties in every app manifest and could confuse contributors. I think that this is a part that they're will not need to dive in. I think we should keep it transparent, and explain the process in documentation as the json format does not allows us to comment.

Reply:
Ok then, I remove it, thanks 👍